packages/reporter/src/test/test.cy.tsx

```diff
@@ -16,7 +16,11 @@ describe('test/test.tsx', () => {
     }
 
     cy.mount(<div className="runnable suite">
-      <Test model={model} appState={appState} />
+      <Test
+        model={model}
+        appState={appState}
+        experimentalStudioEnabled={false}
+      />
     </div>)
 
     cy.percySnapshot()
```

Comment:
I'd expect tests added for when studio is enabled. When is that work planned if it's not happening here? There are quite a skipped tests in the reporter atm for studio. Here's a few I know off-hand

- https://github.com/cypress-io/cypress/blob/develop/packages/reporter/cypress/e2e/runnables.cy.ts#L174
- https://github.com/cypress-io/cypress/blob/develop/packages/reporter/cypress/e2e/tests.cy.ts#L137
- https://github.com/cypress-io/cypress/blob/develop/packages/reporter/cypress/e2e/suites.cy.ts#L133

Reply:
Oh I did not realize we had these skipped - I'll go back and add them in now in https://github.com/cypress-io/cypress/issues/23461

Reply:
Ok un-skipped.

I will write E2E tests in a separate PR once the UI stuff is done, and you'll be tagged to review 💯 

https://github.com/cypress-io/cypress/pull/23413/commits/9c24014b00a42125ae4209a224fa6ab5db97ec20
https://github.com/cypress-io/cypress/pull/23413/commits/c7d66f5ab140f171b126b7cf10a804317267e42e